datasets/has_part/has_part.py

```diff
@@ -0,0 +1,120 @@
+
+from __future__ import absolute_import, division, print_function
+
+import ast
+from collections import defaultdict
+
+import datasets
+
+
+_CITATION = """\
+@misc{bhakthavatsalam2020dogs,
+      title={Do Dogs have Whiskers? A New Knowledge Base of hasPart Relations},
+      author={Sumithra Bhakthavatsalam and Kyle Richardson and Niket Tandon and Peter Clark},
+      year={2020},
+      eprint={2006.07510},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """\
+This dataset is a new knowledge-base (KB) of hasPart relationships, extracted from a large corpus of generic statements. Complementary to other resources available, it is the first which is all three of: accurate (90% precision), salient (covers relationships a person may mention), and has high coverage of common terms (approximated as within a 10 year old’s vocabulary), as well as having several times more hasPart entries than in the popular ontologies ConceptNet and WordNet. In addition, it contains information about quantifiers, argument modifiers, and links the entities to appropriate concepts in Wikipedia and WordNet.
+"""
+
+_HOMEPAGE = "https://allenai.org/data/haspartkb"
+
+_LICENSE = ""
+
+
+TSV_ID = "1Ev4RqWcPsLI9rgOGAKh-_dFKqcEZ1u-G"
+FOLDER_ID = "1NzjXX46NnpxtgxBrkBWFiUbsXAMdd-lB"
+ID = TSV_ID
+
+_URL = f"https://drive.google.com/uc?export=download&id={ID}"
+
+
+class HasPart(datasets.GeneratorBasedBuilder):
+    def _info(self):
+        features = datasets.Features(
+            {
+                "arg1": datasets.features.Value("string"),
+                "arg2": datasets.features.Value("string"),
+                "score": datasets.features.Value("float64"),
+                "wikipedia_primary_page": datasets.features.Sequence(datasets.features.Value("string")),
+                "synset": datasets.features.Sequence(datasets.features.Value("string")),
+            }
+        )
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+
+        dl_fp = dl_manager.download_and_extract(_URL)
+        # input_fp = os.path.join(os.path.dirname(dl_fp), "hasPartKB.tsv")
+        # os.rename(dl_fp, input_fp)
```

Comment:
can you remove the comments please ?

Reply:
Done.  The README already has descriptions of each data field (lines 89 - 92) in the Data Fields section.